build_windows/ChangeLog.txt

```diff
@@ -1,4 +1,8 @@
 
+2024-05-13  David Declerck <david.declerck@ocamlpro.com>
+
+	* vs20xx/libcob.vcxproj[.filters]: add missing profiling.c

```

Comment:
we don't use wildcards, as this is quite global, the "global" entry as below would be fine